bin/node-template/Cargo.toml

```diff
@@ -2,13 +2,13 @@
 name = "node-template"
 version = "2.0.0"
 authors = ["Anonymous"]
-build = "build.rs"
 edition = "2018"
 license = "Unlicense"
+build = "node/build.rs"
 
 [[bin]]
 name = "node-template"
-path = "src/main.rs"
+path = "node/main.rs"
```

Comment:
Why do we not follow default rust project structure?

Reply:
The reason is to make our intention clear. `/src` is where rust developers would quickly focus on and regarded as their main workspace. But in node template, I believe node is not their focus, but `/pallets` should be. We have a `pallet-template` to demonstrate what a simplest pallet would look like. They can play with this pallet template, or extend to write additional pallets and integrate it in the runtime.

Also this new naming make it clear there are `node`, `runtime` that is under node, and (possibly) multiple pallets.

I will confirm on the release script.

Reply:
Node needs also to be modified when you change something on the chain spec for example.

The structure I imagine:
node-template
   - node
       - src
       - Cargo.toml
   - runtime
       - src 
       - Cargo.toml
       - build.rs
   - pallets
       - template
          - src
   - Cargo.toml 

People already always complain that we don't follow default Rust etc and in this case  don't see the reason to not follow default Rust.

Reply:
Cool. Working on the release script so it replace the path in pallet template